### ISSUE TITLE
Exception: NoClassDefFoundError

### ISSUE THREAD

**Issue by dotancohen** [vsch/idea-multimarkdown#387]
Markdown Navigator is throwing an exception when used with **PhpStorm EAP 2017.1**, but I'm not sure how to reproduce it. I'll simply find the exception in the IDE log without knowing exactly what happened to cause it.

This is the IDE:
```
PhpStorm 2017.1 EAP
Build #PS-171.3019.12, built on February 8, 2017
PhpStorm EAP User
Expiration date: March 10, 2017
JRE: 1.8.0_112-release-b702 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 3.10.0-514.2.2.el7.x86_64
```

Here is the stack trace in the hope that it will be useful:
```
com/intellij/lang/java/JavaLanguage
java.lang.NoClassDefFoundError: com/intellij/lang/java/JavaLanguage
	at com.vladsch.idea.multimarkdown.language.completion.FlexmarkJavaCompletionContributor.<init>(FlexmarkJavaCompletionContributor.kt:32)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.picocontainer.defaults.InstantiatingComponentAdapter.newInstance(InstantiatingComponentAdapter.java:193)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.doGetComponentInstance(CachingConstructorInjectionComponentAdapter.java:103)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.instantiateGuarded(CachingConstructorInjectionComponentAdapter.java:80)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.getComponentInstance(CachingConstructorInjectionComponentAdapter.java:63)
	at com.intellij.openapi.extensions.AbstractExtensionPointBean.instantiate(AbstractExtensionPointBean.java:75)
	at com.intellij.openapi.extensions.CustomLoadingExtensionPointBean.instantiateExtension(CustomLoadingExtensionPointBean.java:47)
	at com.intellij.lang.LanguageExtensionPoint.access$000(LanguageExtensionPoint.java:28)
	at com.intellij.lang.LanguageExtensionPoint$1.compute(LanguageExtensionPoint.java:42)
	at com.intellij.openapi.util.NotNullLazyValue.getValue(NotNullLazyValue.java:39)
	at com.intellij.lang.LanguageExtensionPoint.getInstance(LanguageExtensionPoint.java:53)
	at com.intellij.codeInsight.completion.CompletionPreloader.preload(CompletionPreloader.java:32)
	at com.intellij.openapi.application.Preloader.a(Preloader.java:74)
	at com.intellij.openapi.progress.impl.CoreProgressManager$3.run(CoreProgressManager.java:181)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:587)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:532)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:66)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:166)
	at com.intellij.openapi.application.Preloader.a(Preloader.java:72)
	at com.intellij.util.concurrency.BoundedTaskExecutor$2.run(BoundedTaskExecutor.java:212)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.ClassNotFoundException: com.intellij.lang.java.JavaLanguage PluginClassLoader[com.vladsch.idea.multimarkdown, 2.3.0]
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:65)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 27 more
```

**Comment by vsch** [comment]
@dotancohen, it was a bug that was fixed in version 2.3.1.

**Comment by dotancohen** [comment]
I see, thank you. I've upgraded and do not see the issue any longer.
